recipe-client-addon/test/browser/browser_NormandyApi.js

```diff
@@ -15,7 +15,23 @@ add_task(function* () {
   });
 
   // Test that NormandyApi can fetch from the test server.
-  const response = yield NormandyApi.get("browser/test_server.sjs");
-  const data = yield response.json();
-  Assert.deepEqual(data, {test: "data"}, "NormandyApi returned incorrect server data.");
+  let response = yield NormandyApi.get("browser/test_server.sjs");
+  let data = yield response.json();
+  Assert.deepEqual(data, {queryString: {}, body: {}}, "NormandyApi returned incorrect server data.");
```

Comment:
#510 doesn't use any `.sjs` files, but you should  be able to, according to [the httpd docs](https://developer.mozilla.org/en-US/docs/Mozilla/httpd.js/HTTP_server_for_unit_tests#SJS:_server-side_scripts). You might have to add `server.registerContentType("sjs", "sjs");`, but I wouldn't be surprised if that were the default.